src/query/query_parser/query_parser.rs

```diff
@@ -511,6 +591,31 @@ mod test {
              Term([0, 0, 0, 0, 98])]\"",
             false,
         );
+        test_parse_query_to_logical_ast_helper(
+            "title:[a TO b]",
+            "(Included(Term([0, 0, 0, 0, 97])) TO \
+             Included(Term([0, 0, 0, 0, 98])))",
```

Comment:
Possible Silly Q: Is it not possible to test the input text against the actual struct's? Rather than a string interpretation? I ask, because refactoring breaks would be caught by the compiler rather than at test run. That is all.

Reply:
Agreed that this would be great.